src/overlays/gamestates/ovl_title/z_title.c

```diff
@@ -130,7 +132,6 @@ void ConsoleLogo_Main(GameState* thisx) {
 
     if (OOT_DEBUG && gIsCtrlr2Valid) {
         Gfx* gfx = POLY_OPA_DISP;
-        s32 pad;
 
         ConsoleLogo_PrintBuildInfo(&gfx);
```

Comment:
It's a little weird that this function doesn't exist in retail, yet it is called here and expected to be optimized out. Maybe this block makes more sense as `#if OOT_DEBUG` rather than `if (OOT_DEBUG && ...)`?

Reply:
maybe. I didnt put that OOT_DEBUG there, i think you did when you added the define :D
but yeah i can change it. I think as-is it introduces a warning anyway cause there is no function declaration to reference

Reply:
I was going to ask if cc_check were complaining about it.
Probably better to do a `#if OOT_DEBUG` instead

Reply:
oh, whoops